src/Api/include/Api/Orbit.h

```diff
@@ -323,7 +324,7 @@ inline bool orbit_api_active() {
 }
 
 #define ORBIT_CALL(function_name) \
-  if (orbit_api_active() && g_orbit_api_v0.function_name) g_orbit_api_v0.function_name
+  !(orbit_api_active() && g_orbit_api_v0.function_name) ? (void)0 : g_orbit_api_v0.function_name
```

Comment:
question, can you incorporate function call inside the macro?

something like 
```
ORBIT_CALL(function_name, ...)
do { 
  if (...) function_name(__VA_ARGS__);
} while ...
```

Reply:
That would be my thought, too.